src/components/volunteer/VolunteerEventRecord.tsx

```diff
@@ -99,7 +99,7 @@ export const VolunteerEventRecord = ({
         )}
         {device.platform === 'ios' && <TitleShadow />}
 
-        {!!attending?.length && (
+        {isAttendingEvent !== undefined && !!attending?.length && (
```

Comment:
why did you choose `isAttendingEvent !== undefined` here? can we get the same result with `!!isAttendingEvent`?

Reply:
we need to check if `isAttendingEvent` is not set, which means its neither `true` or `false`. if it is `true` or `false` their was a successful validation of the attending status, otherwise we do not have a logged in user, so we will not show any buttons.